plugin/indent_blankline.vim

```diff
@@ -31,6 +31,7 @@ lua require("indent_blankline").init()
 
 augroup IndentBlanklineAutogroup
     autocmd!
+    autocmd OptionSet list lua require("indent_blankline.utils").list_changed()
```

Comment:
you don't need this `list_changed` function, just  add it to the other `OptionSet` autocommand. `IndentBlanklineRefresh` will do the right thing